components/Layout.tsx

```diff
@@ -1,15 +1,12 @@
 import { FC, PropsWithChildren } from 'react';
 import Sidebar from './SideBar';
 import NavBar from './NavBar';
-import { useUser } from '@auth0/nextjs-auth0';
 
 const Layout: FC<PropsWithChildren> = ({ children }) => {
-  const { user } = useUser();
-
   return (
     <>
       <NavBar />
-      {user && <Sidebar />}
+      <Sidebar />
```

Comment:
I think this is removing the conditional rendering you added to this file earlier, maybe revoke this.